tests/test_tools.py

```diff
@@ -388,25 +387,6 @@ def forward(self, location, celsius: str) -> str:
         assert "Nullable" in str(e)
 
 
-@pytest.fixture
-def mock_server_parameters():
-    return MagicMock()
-
-
-@pytest.fixture
-def mock_mcp_adapt():
-    with patch("mcpadapt.core.MCPAdapt") as mock:
-        mock.return_value.__enter__.return_value = ["tool1", "tool2"]
-        mock.return_value.__exit__.return_value = None
-        yield mock
-
-
-@pytest.fixture
-def mock_smolagents_adapter():
-    with patch("mcpadapt.smolagents_adapter.SmolAgentsAdapter") as mock:
-        yield mock
-
-
```

Comment:
I need these mocks for my unit test.

I think we can keep both tests: my unit and your integration test.

Reply:
sure let's put it back